package.json

```diff
@@ -70,10 +70,6 @@
     "gatsby-plugin-sitemap": "^5.19.0",
     "prettier": "^2.7.1"
   },
-  "keywords": [
```

Comment:
why not keep keywords, but update them to applicable ones?

Reply:
Because that's a field for npm packages and npm search but this is not an npm module.